server/apis/links-api.js

```diff
@@ -0,0 +1,43 @@
+/* eslint new-cap: [2, {"capIsNewExceptions": ["Router"]}] */
+const Link = require('../models/links.js');
+const router = require('express').Router();
+// These handle all of the requests to the database.
+
+router.get('/', (req, res) => {
+  Link.read()
+   .then((links) => {
+     res.json({ links });
+   })
+   .catch((error) => {
+     console.log('ERROR: ', error);
+   });
+});
+
+router.delete('/:id', (req, res) => {
+  Link.remove(1) // req.body.id
+  .then((link) => {
+    res.json({ link });
+  })
+  .catch((error) => {
+    console.log('ERROR: ', error);
+  });
+});
+
+router.put('/:id', (req, res) => {
+  Link.update({ id: 1, url: 'youtube.com/ola' }) // req.body.id
```

Comment:
Can we not hardcore the id here, and actually get it out of the route use req.params.id?